variables.tf

```diff
@@ -452,6 +452,30 @@ variable "hms_rw_k8s_pdb_settings" {
   }
 }
 
+variable "hms_ecs_metrics_readwrite_namespace" {
+  description = "ECS readwrite metrics namespace"
+  type        = string
+  default     = "hmsreadwritelegacy"
+}
+
+variable "hms_ecs_metrics_readonly_namespace" {
+  description = "ECS readonly metrics namespace"
+  type        = string
+  default     = "hmsreadonlylegacy"
```

Comment:
`hms_readonly_legacy` was the old value right? same for readwrite.

Reply:
yes changed it to have _